test/state/state.cpp

```diff
@@ -135,6 +135,17 @@ std::variant<int64_t, std::error_code> validate_transaction(const Account& sende
     return execution_gas_limit;
 }
 
+namespace
+{
+void clear_empty_accounts(State& state)
```

Comment:
I think "clear" is not very good name. Let's call it "delete" or "erase". We can consult Yellow Paper.

Reply:
Let's use "delete". 
![Uploading image.png…]()